automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/RegistrationSpec.scala

```diff
@@ -59,6 +60,36 @@ class RegistrationSpec extends FreeSpec with BeforeAndAfter with Matchers with W
     }
   }
 
+  "should show billing account instructions for a newly registered user" in withWebDriver { implicit driver =>
+
+    signIn(email, password)
+    val registrationPage = await ready new RegistrationPage
+
+    registerCleanUpForDeleteUser(subjectId)
+
+    registrationPage.register(
+      firstName = "Test",
+      lastName = "Dummy",
+      title = "Tester",
+      contactEmail = Some("test@firecloud.org"),
+      institute = "Broad",
+      institutionalProgram = "DSDE",
+      nonProfitStatus = true,
+      principalInvestigator = "Nobody",
+      city = "Cambridge",
+      state = "MA",
+      country = "USA")
+
+    await ready new DataLibraryPage()
+
+    val listPage = new WorkspaceListPage()
+    listPage.open
```

Comment:
Combine this into:
```scala
val listPage = new WorkspaceListPage().open
```
And I think the `validateLocation()` below is redundant then.